R/UK.R

```diff
@@ -0,0 +1,519 @@
+#' United Kingdom Class for downloading, cleaning and processing notification
+#' data.
+#'
+#' @description Country specific information for downloading, cleaning
+#'  and processing covid-19 region data for an example Country.
+#'
+#' @details Inherits from `DataClass`
+#' @source https://coronavirus.data.gov.uk/details/download #nolint
+#' @examples
+#' \dontrun{
+#' region <- Italy$new(verbose = TRUE, steps = TRUE)
+#' region$download()
+#' region$clean()
+#' region$process()
+#' region$return()
+#' }
+Uk <- R6::R6Class("Uk", # rename to country name
```

Comment:
all caps for acronyms (as for ECDC etc)

Reply:
Had to add to `check_country_available()` a check that if country name is "Uk", change to "UK" as the country name parser makes the first letter upper case and the others lower case before loading into get.